src/types/lint.ts

```diff
@@ -25,6 +25,8 @@ export interface TemplateLintMessage {
   message: string;
   line: number;
   column: number;
+  endLine: number;
+  endColumn: number;
```

Comment:
```suggestion
  endLine?: number;
  endColumn?: number;
```

Reply:
I think the idea here is that we require a certain version of ember-template-lint, which is the only reason why we would need this.

Reply:
I'm actually going to accept this, which will help for any libraries that want to implement the todos outside of the ones we currently support.